src/cljs/athens/effects.cljs

```diff
@@ -33,7 +33,8 @@
   :reset-conn!
   (fn [new-db]
     (d/reset-conn! db/dsdb new-db)
-    (common-db/health-check db/dsdb)))
+    (when-not (= new-db common-db/empty-db)
```

Comment:
This does nothing, since a health check over a wiped db is instant. I need to avoid the health check on db-dump transact instead.

Reply:
fixed